File: test/e2e-fixtures/validate.py
#!/usr/bin/env python3
import os
import sys

last_message = os.environ.get('AI_LAST_MESSAGE', '')

if not last_message:
    print('No AI response received')
    sys.exit(0)

keywords = ['file', 'directory', 'hello', 'hi', 'test', 'workflow', '.md', '.py', '.js']
if any(keyword.lower() in last_message.lower() for keyword in keywords):
    print('true')
else:
    print(f'Expected keywords but got: {last_message[:200]}...')

File: test/e2e-fixtures/validate.js
#!/usr/bin/env node
const lastMessage = process.env.AI_LAST_MESSAGE || '';

if (!lastMessage) {
  console.log('No AI response received');
  process.exit(0);
}

const keywords = ['file', 'directory', 'hello', 'hi', 'test', 'workflow', '.md', '.py', '.js'];
const hasKeyword = keywords.some((k) => lastMessage.toLowerCase().includes(k.toLowerCase()));

if (hasKeyword) {
  console.log('true');
} else {
  console.log(`Expected keywords but got: ${lastMessage.slice(0, 200)}...`);
}
